tests/conftest.py

```diff
@@ -99,6 +99,25 @@ def subp_exec(
     return stdout, stderr, proc.returncode
 
 
+def test_args_expansion(
+    pth=None, cmd=None, appendix=list(), dry=True
+) -> Tuple[bytes, bytes, int]:
+    """
```

Comment:
can you add a sentence as to what this is for? I can't infer why we need this

Reply:
Done!